n_queen_problem/NQueenProblem.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Arrays;
+
+public class NQueenProblem { 
+    private static String placeQueenOnBoard(int Qi, int[] board) {
+        //base case
+        if (Qi == board.length) {
+            return Arrays.toString(board);
```

Comment:
You need to append a new line with this return statement too, so that each possibility is displayed on a new line.